src/sentry/issues/auto_source_code_config/frame_info.py

```diff
@@ -57,10 +59,6 @@ def transformations(self, frame_file_path: str) -> str:
             is_windows_path = True
             frame_file_path = frame_file_path.replace("\\", "/")
 
-        # Remove leading slash if it exists
```

Comment:
This is the change I want to support in the next PR. I will need to add test cases for this.